app/javascript/packages/document-capture/components/document-capture.tsx

```diff
@@ -133,8 +133,15 @@ function DocumentCapture({ onStepChange = () => {} }: DocumentCaptureProps) {
         },
       ].filter(Boolean) as FormStep[]);
 
+  // If the user got here by opting-in to in-person proofing, when skipDocAuth === 'true',
+  // then set steps to inPersonSteps
+  const steps: FormStep[] = skipDocAuth === 'true' ? inPersonSteps : defaultSteps;
```

Comment:
so `inPersonSteps` here is the opt-in flow, correct?

Reply:
Correct, alone (only the inPersonSteps) is the in person flow. That is what is being used for the opt-in flow. 
The remote flow (defaultSteps) concats with in person flow so IPP is available if id upload fails